### ISSUE TITLE
When refactoring to overlay promises, an error is still given.

### ISSUE THREAD

**Issue by JesperBalslev** [ionic-team/v4-migration-tslint#47]
When changing overlay components (Loading, Toast, Modal, or Alert ) to v4, which are created asynchronously now, the tslint migration still gives an error when you write it like this:

```
showAlert() {
  this.alertCtrl.create({
    message: "Hello There",
    subHeader: "I'm a subheader"
  }).then(alert => alert.present());
}
```
giving this error: 

```
 The create method of overlay controllers now returns a promise.
Please ensure that you are handling this promise correctly.
```

**Comment by dwieeb** [comment]
Thanks @JesperBalslev. We should probably just make that a warning, not an error.

I think if you do this, it may not complain:

```ts
async showAlert() {
  const alert = await this.alertCtrl.create({
    message: "Hello There",
    subHeader: "I'm a subheader"
  });

  await alert.present();
}
```

We just need to add logic to check for a `.then()` call as well

**Comment by JesperBalslev** [comment]
Thanks for the reply, that is true. :)
I just prefer to write it the other way... unless there is a reason i shouldn't?

**Comment by dwieeb** [comment]
No reason, it's a preference! We should handle both ways. 😄

**Comment by godenji** [comment]
Linter output is kind of noisy with these false positives...
